src/nvim/ex_docmd.c

```diff
@@ -1335,25 +1419,108 @@ static char_u * do_one_cmd(char_u **cmdlinep,
    * is equal to the lower.
    */
 
+  if (!IS_USER_CMDIDX(ea.cmdidx)) {
+    if (ea.cmdidx != CMD_SIZE && ea.cmdidx >= 0) {
```

Comment:
I added that because the build complained about this. Maybe something else changed in the meanwhile, because I don't see that issue anymore when I remove it.